rustls/src/msgs/fragmenter.rs

```diff
@@ -25,9 +26,9 @@ impl MessageFragmenter {
     /// Payloads are borrowed.
     pub fn fragment_message<'a>(
         &self,
-        msg: &'a PlainMessage,
+        msg: &'a OutboundOpaqueMessage,
```

Comment:
This is not really the correct type; fragmenting doesn't take a potentially encrypted outgoing message and make definitely-plaintext outgoing message from it. I think it should be `OutboundPlainMessage`?